### DISCUSSION TITLE
建了个微信群用于交流

### DISCUSSION THREAD

**Discussion by zzwtop1** [PaddlePaddle/PaddleNLP#6744] [General]
自己建一个群
![c600c7f4983802c889225acc9e8e031](https://github.com/PaddlePaddle/PaddleNLP/assets/102272958/19b992d4-cd54-4152-884a-8aaae4ce66b6)

**Comment by simon7073** [comment]
二维码过期咯~可以再放一个吗？

**Reply by zzwtop1** [reply]
放好了
